test/Microsoft.AspNetCore.Sockets.Tests/SocketsTestHost.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Sockets.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Sockets.Tests
+{
+    public static class SocketsTestHost
```

Comment:
Should this be in `HttpConnectionDispatcherTests.cs`? It seems to be used only there.